iChef-WebApi/src/main/java/com/kitchen/iChef/Service/RecipeService.java

```diff
@@ -198,6 +199,14 @@ public RecipeDTO updateRecipe(RecipeDTO recipeDTO) {
         throw new ResourceNotFoundException("Not yet implemented.");
     }
 
+    public List<RecipeDTO> sortRecipes(String field, boolean ascending) {
+        List<Recipe> sortedRecipes = new ArrayList<>();
+        Iterable<Recipe> recipes = recipeRepository.findAll(ascending ? Sort.by(field).ascending() : Sort.by("title").descending());
```

Comment:
why are we sorting descending only by title?

`Iterable<Recipe> recipes = recipeRepository.findAll(ascending ? Sort.by(field).ascending() : Sort.by(field).descending());`